### PR TITLE
Docs: Minor spec typos

### PR DESCRIPTION
**Pull request by chellmuth**
Makes a few fixes to osl-languagespec.pdf

### CODE REVIEW

File: src/doc/languagespec.tex [hunk 1]
```diff
@@ -5223,7 +5223,7 @@ \subsection{Light emission closures}
 directly seen by the camera will directly reproduce the closure weight in the
 final pixel, regardless of being a surface or a volume.
 
-For surface emission, If you divide {\cf emission} by 
+For surface emission, if you divide {\cf emittance} by
```
Comment (aconty) [inline src/doc/languagespec.tex:5226]: I think emission here refers to the closure.

Comment (lgritz) [reply src/doc/languagespec.tex:5226]:
```suggestion
For surface emission, if you divide the return value of {\cf emission()} by
```

Comment (lgritz) [reply src/doc/languagespec.tex:5226]: Yeah, I think that removes some of the ambiguity

**Review by lgritz — CHANGES_REQUESTED** [review]
LGTM except for that same place that Alex pointed to. I made a suggestion that I think clarifies the language.

File: src/doc/languagespec.tex [hunk 2]
```diff
@@ -5223,7 +5223,7 @@ \subsection{Light emission closures}
 directly seen by the camera will directly reproduce the closure weight in the
 final pixel, regardless of being a surface or a volume.
 
-For surface emission, If you divide {\cf emission} by 
+For an emissive surface, if you divide the return value of {\cf emission()} by
```
Comment (lgritz) [inline src/doc/languagespec.tex:5226]: Sorry, I didn't realize which section this was in. Further suggestion:

```suggestion
For an emissive surface, if you divide the return value of {\cf uniform_edf()} by
```

File: src/doc/languagespec.tex [hunk 3]
```diff
@@ -5563,7 +5563,7 @@ \subsubsection{Deprecated Emission closures}
 directly seen by the camera will directly reproduce the closure weight in the
 final pixel, regardless of being a surface or a volume.
 
-For surface emission, If you divide {\cf emission()} by 
+For surface emission, if you divide {\cf emission()} by
```
Comment (lgritz) [inline src/doc/languagespec.tex:5566]: Oops, another spot here:

```suggestion
For an emissive surface, if you divide the return value of {\cf emission()} by
```

**Pull request merged**